udata_tabular_preview/preview.py

```diff
@@ -36,8 +34,7 @@ def can_preview(self, resource):
             not max_size
             or (extras_size or resource.filesize or float("inf")) <= max_size
         )
-
         return all((has_config, is_supported, is_allowed, size_ok))
 
     def preview_url(self, resource):
-        return f'{self.preview_base_url}/?url={quote_plus(resource.latest)}'
+        return f'{self.preview_base_url}/resources/{resource.id}'
```

Comment:
Shouldn't it be `/datasets/{dataset.id}/resources/{resource.id}`?

Reply:
it was difficult to get dataset so I developped another route in explore so no, this route is correct and will redirect to the right exploration. But I think error is due to a mistake in `udata__explore_url` param in deployment